fhir-converter/docker-entrypoint.sh

```diff
@@ -0,0 +1,18 @@
+#!/bin/bash
+if [[ $OSTYPE == 'darwin'* ]]; then
+  FREE_FINAL=$(( $(sysctl -a | awk '/memsize/{print $2}') / 2**20 ));
+else
+  FREE=$(awk '/MemFree/ { printf "%.3f \n", $2/1024 }' /proc/meminfo)
+  FREE_FINAL=${FREE%.*}
+fi
+FREE_IDEAL=$(($FREE_FINAL-512))
+if [[ FREE_IDEAL -lt 128 ]]
+then
+  MEM_LIMIT=128
+else
+  MEM_LIMIT=$(($FREE_IDEAL))
+fi
+export MEMORY_LIMIT="${MEM_LIMIT:=4096}";
+export NODE_OPTIONS="--max-old-space-size=${MEMORY_LIMIT}"
+echo "NODE_OPTIONS: ${NODE_OPTIONS}"
+npm start
```

Comment:
nit: some comments about what this is doing would be useful for future reference